india_compliance/income_tax_india/overrides/test_asset_depreciation_schedule.py

```
@@ -0,0 +1,317 @@
+import frappe
+from erpnext.accounts.utils import get_fiscal_year
+from erpnext.assets.doctype.asset_depreciation_schedule.asset_depreciation_schedule import (
+    get_asset_depr_schedule_doc,
+)
+from frappe.tests import IntegrationTestCase
+from frappe.utils import add_days, date_diff, flt, getdate
```

Flake8 (7.3.0)

[error] 'frappe.utils.getdate' imported but unused

(F401)